src/Dhall/JSON.hs

```diff
@@ -192,3 +198,27 @@ omitNull (Bool bool) =
     Bool bool
 omitNull Null =
     Null
+
+{-| Convert a piece of Text carrying a Dhall inscription to an equivalent JSON Value
+
+>>> :set -XOverloadedStrings
+>>> :set -XOverloadedLists
```

Comment:
I think this example doesn't require `-XOverloadedLists`

Reply:
Done.